Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -1275,7 +1275,8 @@ def parse_incident_from_item(item, is_fetch):  # pragma: no cover
                                 # save attachment to incident
                                 incident['attachment'].append({
                                     'path': file_result['FileID'],
-                                    'name': get_attachment_name(attachment.name)
+                                    'name': get_attachment_name(attachment.name),
+                                    "description": "is_attached_file" if not attachment.is_inline else ""
```

Comment:
@mmhw please move it to CSP and make it an Enum (the same goes for all other occurrences)

Reply:
Addad